docs/sources/whatsnew/whats-new-next/README.md

```diff
@@ -0,0 +1,57 @@
+# Contribute to 'What's New in Grafana Cloud'
+
+To have a feature presented in [What's New in Grafana Cloud](https://grafana.com/docs/grafana-cloud/whatsnew/), add an entry to the [`index.md`](./index.md) file in this directory.
+
+Use the following template, replace any `<VARIABLE>` with the appropriate text, explained later:
+
+```markdown
+## <FEATURE>
+
+<!-- <CONTRIBUTOR> -->
+<!-- <AVAILABILITY> -->
+
+<CLOUD AVAILABILITY> in Grafana <CLOUD OFFERING>
+
+<DESCRIPTION>
+```
+
+## _`CONTRIBUTOR`_
+
+The name of the contributor of the feature.
+The information is intentionally commented out so that it isn't displayed in the published page.
+
+## _`AVAILABILITY`_
+
+One of:
```

Comment:
Is the assumption here:

- If it's in OSS, it's also in Enterprise
- If someone just writes Enterprise here, they mean it's not in OSS

I wonder if it's just easier and less potentially confusing for all involved (including the person who has to parse this for the OSS What's new) to just have folks list all of the on-prem versions it's available in rather than having to parse an exclusionary  statement.
```suggestion
One or both of:
```